recipe/migrations/pypy38.yaml

```diff
@@ -0,0 +1,52 @@
+migrator_ts: 1647123563
+__migrator:
+    migration_number: 1
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
```

Comment:
Do we still need 3.6 here?

Reply:
I believe that's just for the ordering, cf. https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2601/commits/2f37b122c6efe02d957af623d5d00fea38d4e31e